src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -91,7 +92,7 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
     return <div>{error.message}</div>;
   }
   return (
-    <>
+    <ErrorBoundary>
```

Comment:
This is probably a generalized question, but when do you want to put the error boundary around a specific component like this, vs the entire app?

Reply:
ErrorBoundaries can be used anywhere an error should be contained - so they can and will be used around the entire app as well as around specific components.